doc/source/ray-contribute/development.rst

```diff
@@ -45,27 +101,30 @@ Running the script will remove the  ``ray/tune``, ``ray/rllib``, ``ray/autoscale
     # with your local `ray/python/ray/<package>`.
     python python/ray/setup-dev.py
 
-.. warning:: Do not run ``pip uninstall ray`` or ``pip install -U`` (for Ray or Ray wheels) if setting up your environment this way. To uninstall or upgrade, you must first ``rm -rf`` the pip-installation site (usually a ``site-packages/ray`` location), then do a pip reinstall (see 1. above), and finally run the above `setup-dev.py` script again.
+.. warning:: Do not run ``pip uninstall ray`` or ``pip install -U`` (for Ray or Ray wheels) if setting up your environment this way. To uninstall or upgrade, you must first ``rm -rf`` the pip-installation site (usually a directory at the ``site-packages/ray`` location), then do a pip reinstall (see the command above), and finally run the above ``setup-dev.py`` script again.
 
 .. code-block:: shell
 
     # To uninstall, delete the symlinks first.
     rm -rf <package path>/site-packages/ray # Path will be in the output of `setup-dev.py`.
     pip uninstall ray # or `pip install -U <wheel>`
 
-Building Ray on Linux & MacOS (full)
-------------------------------------
+Preparing to build Ray on Linux
+-------------------------------
 
 .. tip:: If you are only editing Tune/RLlib/Autoscaler files, follow instructions for :ref:`python-develop` to avoid long build times.
 
-To build Ray, first install the following dependencies.
```

Comment:
I find "the following" to be clearer. Maybe settle for:

`To build Ray on Ubuntu, run the following commands:`

to avoid the clumsy duplication of the word "following" (if that's the concern).

Reply:
Sure thing! Done.